kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js

```diff
@@ -421,18 +421,19 @@ define([
 
                     widgets.push(widget);
                     return widget.start({
-                        node: document.getElementById(parameterInfo.view[paramSpec.id].id),
+                        node: container.querySelector('#' + parameterInfo.view[paramSpec.id].id),
```

Comment:
prevent possible test clashes by deleting the element within the container, rather than the document, where there could be several elements with the same ID due to the fact that all tests share the same document